docs/pages/reference/cli.mdx

```diff
@@ -276,6 +276,78 @@ Run `tsh help <subcommand>` or see the [Global Flags section](#tsh-global-flags)
 $ tsh --user=joe --login=ec2-user join <session-id>
 ```
 
+### tsh recordings ls
+
+List recorded sessions.
+
+```code
+$ tsh recordings ls [<flags>]
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--from-utc` | 24 hours ago | date | Start of time range in which recordings are listed. Format 2006-01-02. Defaults to 24 hours ago. |
+| `--to-utc` | current | date | Start of time range in which recordings are listed. Format 2006-01-02. Defaults to 24 hours ago. |
+| `--limit` | 50 | number | Maximum number of recordings to show. |
+| `--last` | none | duration | Duration into the past from which session recordings should be listed. Format 5h30m40s |
+
+#### Global flags
```

Comment:
Not a question for this PR, but I think it's silly that we duplicate this section for every single command.

cc @alexfornuto @ptgott

Reply:
Agreed. I haven't been around long enough to know what convention or conversation (if any) led to this being the case, but I think it's worth revisiting.